d2js/js/README.md

```diff
@@ -56,6 +56,11 @@ Compiles D2 markup into an intermediate representation.
 Options:
 - `layout`: Layout engine to use ('dagre' | 'elk') [default: 'dagre']
 - `sketch`: Enable sketch mode [default: false]
+- `themeId`: Theme ID to use [default: 0]
+- `darkThemeId`: Theme ID to use when client is in dark mode
```

Comment:
let's use the [capitalization in the codebase](https://github.com/alixander/d2/blob/73f2588f8cbc29c045eb636ddeefd3d4913a6bf3/d2renderers/d2svg/d2svg.go#L79), which is `ID` instead of `Id`